model/worker.go

```diff
@@ -6,7 +6,7 @@ type Worker struct {
 	WorkerID string `gorm:"primaryKey"`
 	Host     string
 	Status   string
-	Workload int // How much the worker has to do. +1 per silence detection job, +2 per converting job, +3 per streaming job
+	Workload uint // How much the worker has to do. +1 per silence detection job, +2 per converting job, +3 per streaming job
```

Comment:
This requires a change in 
https://github.com/joschahenningsen/TUM-Live/blob/02400f7406905c59c12731369edc1283e25c19ce/api_grpc/worker.go#L189